prime-router/src/test/kotlin/fhirengine/translation/hl7/FhirToHl7ConverterTests.kt

```diff
@@ -417,12 +419,16 @@ class FhirToHl7ConverterTests {
         assertThat { FhirToHl7Converter(schema).convert(bundle) }.isFailure()
 
         // Use a file based schema which will fail as we do not have enough data in the bundle
-        assertThat {
+        val missingDataEx = assertFailsWith<ConfigSchemaElementProcessingException> {
             FhirToHl7Converter(
                 "ORU_R01",
                 "src/test/resources/fhirengine/translation/hl7/schema/schema-read-test-01"
             ).convert(bundle)
-        }.isFailure()
+        }
+        assertThat(missingDataEx.message).isEqualTo(
+            "Error encountered while applying: ORU_R01/header-message-headers in ORU_R01 to FHIR bundle. \n" +
```

Comment:
This is the test case I was thinking of in my other comment, the child schema in this case is `ORU_R01/header`, and the element name is `message-headers`, but it's hard to tell that from the message alone.

Reply:
Agree, I think this will make debugging harder